frontend_addon/{{ cookiecutter.__folder_name }}/.github/workflows/acceptance.yml

```diff
@@ -5,6 +5,7 @@ on:
       - "*.js"
       - "*.json"
       - "*.yaml"
+      - "*.yml"
```

Comment:
@ericof why `*.yaml`? Did you want to point to `.github/workflows/&.yml`? if so, they are not triggering, I would point each related *.yml in each workflow. What do you think?